api/src/main/java/uk/gov/dhsc/htbhf/dwp/model/EligibilityStatus.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.dhsc.htbhf.dwp.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+@JsonFormat(shape = JsonFormat.Shape.STRING)
+public enum EligibilityStatus {
```

Comment:
Can we have some javadoc explaining this enum?
e.g. "The possible states that a claim can be in according to the DWP", plus in-line comments about each one

Reply:
Updated class with javadoc.